app/bundles/CampaignBundle/Config/config.php

```diff
@@ -57,11 +57,28 @@
         'main' => array(
             'priority' => 10,
             'items'    => array(
-                'mautic.campaign.campaigns' => array(
-                    'route'     => 'mautic_campaign_index',
+                'mautic.campaigns.menu.root' => array(
                     'id'        => 'mautic_campaigns_root',
                     'iconClass' => 'fa-clock-o',
-                    'access'    => 'campaign:campaigns:view'
+                    'access'    => 'campaign:campaigns:view',
+                    'children'  => array(
+                        'mautic.campaign.menu.index'  => array(
+                            'route'     => 'mautic_campaign_index',
+                            'access'    => 'campaign:campaigns:view'
+                        ),
+                        'mautic.form.forms'  => array(
+                            'route'     => 'mautic_form_index',
+                            'access'    =>  array('form:forms:viewown', 'form:forms:viewother')
+                        ),
+                        'mautic.page.pages'  => array(
+                            'route'     => 'mautic_page_index',
+                            'access'    => array('page:pages:viewown', 'page:pages:viewother')
+                        ),
+                        'mautic.asset.assets'  => array(
+                            'route'     => 'mautic_asset_index',
+                            'access'    => array('asset:assets:viewown', 'asset:assets:viewother')
+                        ),
+                    )
```

Comment:
I don't like that form, page and asset menu items are defined in the campaign bundle. But I guess that's the only way how to add them under the campaign menu item :(